lib/l10n/app_en.arb

```diff
@@ -636,5 +636,19 @@
   "@alert_publisher_unknown": {
     "description": "Shown on the alert detail screen when the publisher of an alert is unknown"
   },
-  "failed_to_open_url": "Failed to open URL"
+  "failed_to_open_url": "Failed to open URL",
+  "error_dialog_headline": "Oh no - something went wrong :(",
+  "error_dialog_description": "FOSSWarn has noticed an error. Please report the error and attach the following log.",
+  "error_dialog_privacy": "The log does not contain any privacy-sensitive information besides maybe your selected places. FOSSWarn does not send any logs to a server.",
+  "error_dialog_instruction_headline": "What should I do now?",
+  "error_dialog_instruction_body": "1. Check if the error is just because of a missing internet connection\n2. Check if there is already an open issue on Github\n3. Find the relevant part of the log. If the same error message occurs multiple times in the log, sending just one error message is enough.\n4. Open a new issue or write an email to foss-warn@posteo.de",
```

Comment:
```suggestion
  "error_dialog_instruction_body": "1. Check if the error is just because of a missing internet connection\n2. Check if there is already an open issue on GitHub\n3. Find the relevant part of the log. If the same error message occurs multiple times in the log, sending just one error message is enough.\n4. Open a new issue or write an email to foss-warn@posteo.de",
```